docker/ci/dockerfiles/test.rockylinux8.opensearch-dashboards.x64.arm64.dockerfile

```diff
@@ -66,7 +66,7 @@ WORKDIR /usr/share/opensearch
 # Hard code node version and yarn version for now
 # nvm environment variables
 ENV NVM_DIR /usr/share/opensearch/.nvm
```

Comment:
can we add `jq` support https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/152#issuecomment-1080951494

Reply:
Added now.